.github/workflows/generate.yml

```diff
@@ -59,4 +59,24 @@ jobs:
       - name: E2E-Tests
         working-directory: ${{env.working-directory}}
         run: |
-          SKIP_INSTALL_TESTS=TRUE make e2e-tests
\ No newline at end of file
+          make e2e-tests
```

Comment:
Ignore this file. Before merging this PR will undo changes for this file along with rebase. Wanted to run the whole test suite on the new changes.

Reply:
Note to remember to do this